pkg/cmd/builder_test.go

```diff
@@ -0,0 +1,38 @@
+package cmd
```

Comment:
nit: could you add the license here?

Side note, we could automate the update and verification of licenses at some point. We already did it in metrics-server at some point: https://github.com/kubernetes-sigs/metrics-server/blob/master/Makefile#L192-L208

Reply:
Oops, right. License added.

I'll add a license checker later.